src/translations/screens/Tickets.ts

```diff
@@ -14,17 +14,11 @@ const TicketsTexts = {
     button: {
       single: {
         text: _('Ny enkeltbillett', 'New single ticket'),
-        a11yHint: _(
-          'Aktivér for å kjøpe ny enkeltbillett',
-          'Activate to buy a new single ticket',
-        ),
+        a11yHint: _('Kjøp ny enkeltbillett', 'Buy a new single ticket'),
```

Comment:
Litt usikker på disse endringene, er de ment å være med? Vi har et mønster med å bruke "Aktivèr for ..." på accessibilityHint.

Reply:
Jeg ville hatt med "Aktivèr for ..." nå for å være konsekvent, også får vi heller ta en runde på endring av dette senere. Kanskje  @hildeor, @mikaelbr eller noen andre vet hvor dypt beslutningsgrunnlag som ligger bak det at man bruker "Aktivér for ..." på de forskjellige hintene.

Reply:
ja, det er ok. jeg ruller tilbake "Aktivér for å.."